models/article.models.js

```diff
@@ -47,7 +41,9 @@ exports.updateArticleById = (id, update) => {
 
 exports.fetchArticles = () => {
   return db
-    .query("SELECT * FROM articles ORDER BY created_at desc;")
+    .query(
+      "SELECT articles.author, title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, COUNT(comment_id)::INT AS comment_count FROM articles LEFT JOIN comments ON comments.article_id=articles.article_id GROUP BY articles.article_id, articles.author, title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes ORDER BY articles.created_at desc;"
```

Comment:
Nice! I would apply the same LEFT JOIN technique to your single article endpoint too